api/PclusterApiHandler.py

```diff
@@ -116,8 +118,24 @@ def sigv4_request(method, host, path, params={}, headers={}, body=None):
 
     return req_call(boto_request.url, data=body_data, headers=boto_request.headers, timeout=30)
 
+def refresh_tokens(refresh_token):
+    auth = requests.auth.HTTPBasicAuth(CLIENT_ID, CLIENT_SECRET)
+
+    resp = requests.post(
+        TOKEN_URL,
+        data={"grant_type": 'refresh_token', "refresh_token": refresh_token, "client_id": CLIENT_ID},
+        auth=auth,
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+    )
 
-# Wrappers
+    if resp.status_code != 200:
+        raise RefreshTokenError(resp.json().get('error'))
+
+    values = resp.json()
+    access_token = values.get("access_token")
+    id_token = values.get("id_token")
+
+    return {'accessToken': access_token, 'idToken': id_token}
```

Comment:
question: why do we not expect a new refresh token?(I'm reading [this for reference](https://auth0.com/docs/secure/tokens/refresh-tokens/refresh-token-rotation), and it seems the flow involves returning a new refresh token)

Reply:
Cognito doesn't return a new refresh token (as other OIDC providers do) but instead you need to keep the old one and reuse it until it expires